client/lib/perfmon/index.js

```diff
@@ -0,0 +1,173 @@
+/** 
+ * Internal dependencies
+ */
+import analytics from 'analytics';
+
+/**
+ * External dependencies
+ */
+import each from 'lodash/collection/each';
+import remove from 'lodash/array/remove';
+import debounce from 'lodash/function/debounce';
+import page from 'page';
+
+var debug = require( 'debug' )( 'calypso:perfmon' );
+
+const PLACEHOLDER_CLASSES = [
+	'placeholder',
+	'pulsing-dot is-active',
+	'is-loading'
+];
+
+const PLACEHOLDER_MATCHER = PLACEHOLDER_CLASSES.map(function(clazz) { return `[class*='${clazz}']`; }).join(', ');
+const OBSERVE_ROOT = document.getElementById('wpcom');
```

Comment:
This name here is confusing to me because of the verb. Could it be some noun, such as `OBSERVER_ROOT` or `OBSERVATION_ROOT` or `DOCUMENT_ROOT` or `CONTENT_ROOT`?